src/components/settings/SettingsSSOTab/SSOStepper.jsx

```diff
@@ -65,7 +81,8 @@ const SSOStepper = ({ enterpriseSlug, enterpriseId, enterpriseName }) => {
     if (configValues !== null) {
       configValues.append('enterprise_customer_uuid', enterpriseId);
       try {
-        await LmsApiService.updateProviderConfig(configValues, providerConfig.id);
+        const response = await LmsApiService.updateProviderConfig(configValues, providerConfig.id);
```

Comment:
updating configs would generate a new ID which we never set to the provider config state, which would cause issues down the line